lib/components/InputField/InputField.tsx

```diff
@@ -28,7 +46,10 @@ export const InputField = React.forwardRef<HTMLInputElement, InputFieldProps>(fu
     <>
       <label
         htmlFor={inputId}
-        className="ds-mb-4 ds-inline-block ds-align-top ds-text-form-label ds-font-arial ds-text-black"
+        className={tc([
+          hideLabel ? 'ds-hidden' : '',
+          'ds-mb-4 ds-inline-block ds-align-top ds-text-form-label ds-font-arial ds-text-black',
+        ])}
       >
         {label}
```

Comment:
Looking at the types, if `hideLabel` is true, will this `<label>` element become empty because `label` prop is set to `never`? How does a screenreader work in this situation?

Reply:
If you can double test it with the screenreader to confirm my findings, but screenreader in this situation seems to read the placeholder value.